### PR TITLE
[Android] Account for user's setting of styles property textAllCaps

### LINKED ISSUE #11703: Android textAllCaps no longer works
### Description
Android textAllCaps no longer works when using XF 4.8.0.1269

### Steps to Reproduce

1. Install XF 4.8.0.1269
2. Run Android App
3. Button text is always in capitals

### Expected Behavior
Button text should be camel case

### Actual Behavior
Button text always capitals

### Basic Information

- Version with issue:
- Last known good version: XF 4.7.0.1260
- IDE: VS 2019 V16.7.0
- Platform Target Frameworks: 
  - Android:

### PR DESCRIPTION
**Pull request by rachelkang**
### Description of Change ###
Ensure that user's textAllCaps style property setting is accounted for, and not overridden by default TextTransform property

### Issues Resolved ### 
fixes #11703 

### API Changes ###
 None

### Platforms Affected ### 
Android

### Behavioral/Visual Changes ###
When users set textAllCaps style property to false (specifically for a button), the (button) text will no longer appear in the default ALL CAPS

### Before/After Screenshots ### 

**Before:** 
Button text ("Button text") appears in all caps ("BUTTON TEXT"), regardless of whether or not user sets textAllCaps to false

**After:** 
Button text ("Button text") appears in all caps ("BUTTON TEXT"), when user does not set textAllCaps property

Button text ("Button text") appears in expected casing ("Button text"), when user sets textAllCaps to false

### Testing Procedure ###
This test requires manual review, both on an Android device with API >= 28 and on an Android device with API < 28.
1. In App.cs, switch out `SetMainPage(CreateDefaultMainPage());` for `SetMainPage(Issue11703());`
2. Run on Android, and check that the button text appears in all caps as "BUTTON TEXT". This is the expected default.
3. Now, in styles.xml, comment in the following line: `<item name="android:textAllCaps">false</item>`
4. Run on Android again, and check that the button text no longer appears in all caps, and appears now as "Button text" as expected.
5. Repeat the steps on an Android device with API < 28.

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/xamarin/Xamarin.Forms?pullRequest=12170) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/xamarin/Xamarin.Forms?pullRequest=12170) before we can accept your contribution.<br/><br/>:x: rachelkang [sign now](https://cla.dotnetfoundation.org/xamarin/Xamarin.Forms?pullRequest=12170)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/xamarin/Xamarin.Forms?pullRequest=12170) it.</sub>

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue11703.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.ManualReview)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 11703, "Android textAllCaps no longer works", PlatformAffected.Android)]
+	public class Issue11703 : TestContentPage // or TestMasterDetailPage, etc ...
```
Comment (hartez) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue11703.cs:17]: You can remove this comment: `// or TestMasterDetailPage, etc ...`

**Review by hartez — APPROVED** [review]

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue11703.cs (hunk 1 above)
Comment (rachelkang) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue11703.cs]: removed, thanks @hartez!

**Review by hartez — APPROVED** [review]

File: Xamarin.Forms.Platform.Android/ButtonLayoutManager.cs [hunk 2]
```diff
@@ -273,8 +275,8 @@ bool UpdateTextAndImage()
 				return false;
 
 			var textTransform = _element.TextTransform;
-			
-			_renderer.View.SetAllCaps(textTransform == TextTransform.Default);
```
Comment (PureWeen) [inline Xamarin.Forms.Platform.Android/ButtonLayoutManager.cs:277]: I think what makes the most sense here would be to just have a check like this

```C#
          if(label.TextTransform == Forms.TextTransform.Default)
                textView.TransformationMethod = _defaultTransformationMethod;
            else
                 textView.TransformationMethod = null;
```

**Review by PureWeen — CHANGES_REQUESTED** [review]

File: Xamarin.Forms.Platform.Android/ButtonLayoutManager.cs (hunk 2 above)
Comment (rachelkang) [reply Xamarin.Forms.Platform.Android/ButtonLayoutManager.cs:277]: fixed!

**Review by PureWeen — APPROVED** [review]

**Comment by rmarinho** [comment]
we!! nice work @rachelkang !!

**Pull request merged**